### ISSUE TITLE
Artwork taking a long time to appear in DIscord

### ISSUE THREAD

**Issue by amethystau** [demaisj/soundcloud-rp#29]
Not sure if it's just a problem with me but artwork can take up to 3 minutes to appear in Discord, some not even appearing at all before songs finish.

**Comment by demaisj** [comment]
Hello, double check your upload speed, this might be the problem.

**Comment by amethystau** [comment]
17 Mbps should be enough to handle this process fine ?

**Comment by demaisj** [comment]
Yes that's more than enough, I don't know where the problem could come from then...

**Comment by kuromiii** [comment]
I believe their CDN does not update as fast as it used to anymore...

**Comment by amethystau** [comment]
that sucks, hopefully it just one of things that works itself out eventually.
